libs/platform/src/lib/components/split-menu-button/split-menu-button.component.ts

```diff
@@ -20,6 +21,9 @@ import { BaseComponent } from '../base';
  *
  *
  */
+
+const PRIMARY_BUTTON_MAX_WIDTH_COZY = '9.75rem'; // Cozy 9.75rem. (Max width) 12rem - 2.25rem (menu button width)
+const PRIMARY_BUTTON_MAX_WIDTH_COMPACT = '10rem'; // Compact 10rem. (Max width) 12rem - 2rem (menu button width)
```

Comment:
This approach is completely unnecessary 
![image](https://user-images.githubusercontent.com/26483208/94556980-3e47b400-025e-11eb-8157-4919e53a9b1c.png)

This can be achieved only by adding `max-width: 12rem` to `fd-split-button`

Reply:
fixed.